target/linux/ipq807x/config-5.15

```diff
@@ -440,6 +441,7 @@ CONFIG_SCSI_COMMON=y
 # CONFIG_SDM_GPUCC_845 is not set
 # CONFIG_SDM_LPASSCC_845 is not set
 # CONFIG_SDM_VIDEOCC_845 is not set
+CONFIG_SENSORS_G761=y
```

Comment:
What is this for?

Reply:
Ahh, its the fan driver below.

Reply:
Yeah.. I would not try to upstream this. It just turns the fan on but I have no clue how it works :). 

We could probably even remove it for now. Not major for initial support.

Reply:
Then better remove it

Reply:
66147c8a3b RAX120: Remove unsupported g761 driver